_test/bootstrap.sh

```diff
@@ -8,20 +8,32 @@ fi
 if [ ! -z "$CABALVER" ]; then
     export PATH=/opt/cabal/${CABALVER}/bin:$PATH
 fi
+
+# What directory is this script in?
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+deps=$(
+  cut -d: -f2 "$DIR/dependencies.txt" |
+  xargs echo |
+  tr " " "\n" |
+  sort |
+  uniq |
+  # Exclude packages included in GHC
+  # TODO: This is obviously not ideal
+  # what if the list of packages included with GHC changes?
+  # How can we avoid having to hard-code this list?
+  grep -v 'base\|array\|containers\|time'
+)
+
+if echo "$deps" | grep -q 'old-locale'; then
+  # Move old-locale to the bottom.
+  # Unfortunate, but without this cabal-install tries to update old-locale.
```

Comment:
Maybe I should grab a machine with GHC 7.6 or 7.8 and investigate, because an explanation would be great.